docs/Build-from-Source.md

```diff
@@ -21,5 +21,19 @@ After this command completes, you will find a `cbioportal.war` file suitable
 for Apache Tomcat deployment in `portal/target/`. It is not neccessary to
 install Tomcat yourself, since a command line runnable version of Tomcat is
 provided as a dependency in `portal/target/dependency/webapp-runner.jar`.
+However, if you will be deploying to a standalone Tomcat installation, and
+if you have configured Tomcat to use the Redission client for user session
+management, you should expect a clash between the Redission client being
+used for session management and the Redission client which is embedded in
+the cbioportal.war file for the optional "redis" persitence layer caching
+mode. In this case, you should avoid using the "redis" option for the portal
+property `persistence.cache_type` and you should prevent the Redission
+client from being packaged in cbioportal.war by building with this command
+instead:
```

Comment:
it might be good to elaborate why this doesn't cause a problem when using session management through `webapp-runner.jar`. That also uses redisson as far as I know (https://github.com/heroku/webapp-runner/blob/028d73aa028645757d8e3d383cb81b15e6023bc4/redis/src/main/java/webapp/runner/launch/RedisSessionStore.java). My guess is that there it works fine b/c the redisson dependency is baked into that jar and not into the war 🤷‍♂️?

Reply:
we find this problem perplexing